src/parser/validate.ts

```diff
@@ -54,43 +56,13 @@ export function validate(parse: {
     }
   }
 
-  function validateFlags() {
+  async function validateFlags() {
     for (const [name, flag] of Object.entries(parse.input.flags)) {
```

Comment:
I'd love to see if parallelizing this would help with perf (both the various validations for each flag, AND doing all the flags together)